Formula/fastme.rb

```diff
@@ -0,0 +1,45 @@
+class Fastme < Formula
+  desc "Accurate and fast distance-based phylogeny inference program"
+  homepage "http://www.atgc-montpellier.fr/fastme/"
+  url "https://gite.lirmm.fr/atgc/FastME/repository/v2.1.5.2/archive.tar.gz"
+  sha256 "fc19fb51ba979242eefd5cb12a772b75131fcc44269afec575798b1350a5c57e"
+
+  option "with-openmp", "Enable parallel computations with OpenMP"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+
+  depends_on "gcc" if build.with? "openmp"
+
+  fails_with :clang if build.with? "openmp"
+
+  def install
+    ENV["USE_OPENMP"] = "1" if build.with? "openmp"
+
+    inreplace "configure.ac", "AM_INIT_AUTOMAKE", "AC_CONFIG_MACRO_DIRS([m4])\nAM_INIT_AUTOMAKE"
+
+    system "aclocal"
```

Comment:
Apparently the release tarball is broken: it includes a configure script, but others such as install.sh are not present. This should be reported to them and fixed, rather than depending on autotools and running all those (and patching the `configure.ac`).